recipes/openldap/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "openldap" %}
+{% set version = "2.4.46" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-{{ version }}.tgz
+  sha256: 41975e79406502c092732c57ef0c2c2eb318d91e8e765f81f5d4ab6c1db727c5
+
+build:
+  number: 0
+  skip: True  # [not unix]
+  run_exports:
+    - {{ pin_subpackage('openldap', max_pin='x.x') }}
+  script: 
+    - ./configure --prefix=$PREFIX
+    - make
+    - make install 
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+test:
+  commands:
+    - test $PREFIX/include/ldap.h   # [unix]
+  # imports:
+  #   - ldap
+  #   - ldap.controls
+  #   - ldap.extop
+  #   - ldap.schema
+  #   - slapdtest
+
+about:
+  home: https://www.openldap.org
+  license: The OpenLDAP Public License
+  license_family: OTHER
+  license_file: LICENSE
+  summary: >
+    OpenLDAP Software is an open source implementation of the Lightweight Directory Access Protocol.
+  doc_url: https://www.openldap.org/doc/
+  dev_url: https://www.openldap.org/software/repo.html
+
+extra:
+  recipe-maintainers:
+    - mariusvniekerk
```

Comment:
Happy to be added, if you're looking for :back: :up:

Reply:
done